tests/airport.spec.js

```diff
@@ -149,13 +154,22 @@ console.log(`Test 6 - Check that you can't add planes beyond set airport capacit
 // Arrange
 airport = new Airport();
 plane = new Plane();
+plane2 = new Plane();
+plane3 = new Plane();
+plane4 = new Plane();
+plane5 = new Plane();
+plane6 = new Plane();
 expected = 5;
 
 // Act
 airport.setCapacity(5);
-for (let i = 0; i < 6; i++) {
-    airport.landPlane(plane);
-}
```

Comment:
I honestly wasn't happy switching away from the for loop when refactoring this test that I thought was an elegant way of running my landplane method, but I had a difficult time wrapping my head around being versatile in how I call instances of classes.